src/OSPSuite.UI/Services/DataImporter.cs

```diff
@@ -121,7 +128,8 @@ DataImporterSettings dataImporterSettings
                importerPresenter.LoadConfiguration(configuration);
                using (var importerModalPresenter = _container.Resolve<IModalImporterPresenter>())
```

Comment:
Use _applicationControl to retrieve modal Presenter and don't use the _contaienr directly (might be from old code)

Reply:
done